packages/mobile/src/home/WalletHome.tsx

```diff
@@ -90,12 +96,29 @@ export class WalletHome extends React.Component<Props> {
     this.props.refreshAllBalances()
   }
 
-  componentDidMount() {
+  componentDidMount = async () => {
     // TODO find a better home for this, its unrelated to wallet home
     this.props.initializeSentryUserContext()
     if (SHOW_TESTNET_BANNER) {
       this.showTestnetBanner()
     }
+    await this.tryImportContacts()
```

Comment:
Consider setting a timeout and triggering this after 2 seconds, so it doesn't slow down other things like loading the feed.

Reply:
That may not be necessary. Looks like the first render fills in the tx feed pretty well. You'll be able to notice when the import is done because recipients will populate ~5 seconds later so doesn't appear that anything is being held up

Reply:
I would test it on a shitty phone before to be sure but okay sounds good

Reply:
Opted to use a setTimeout of 0.5 secs as a compromise for now and we can always ramp that up if need be